pkgs/desktops/gnome/core/libgnome-keyring/default.nix

```diff
@@ -1,14 +1,16 @@
-{ lib, stdenv, fetchurl, glib, dbus, libgcrypt, pkg-config, intltool, gobject-introspection, gnome }:
+{ lib, stdenv, fetchurl, glib, dbus, libgcrypt, pkg-config, intltool, gobject-introspection, gnome
+, testers
+}:
 
 let
   pname = "libgnome-keyring";
   version = "3.12.0";
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   name = "${pname}-${version}";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${name}.tar.xz";
+    url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
```

Comment:
Missing `let inherit`?

Reply:
That one is actually not an error, see the let binding above -- it is sort of half converted to use `pname` and `version`. It should be fully converted but in a second PR I think so this one stays no rebuilds.

Reply:
I must have read it too quickly then.